### DISCUSSION TITLE
Welcome to loxone-java Discussions!

### DISCUSSION THREAD

**Discussion by jimirocks** [Smarteon/loxone-java#126] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by TCke83** [comment]
Hi,

i'm Tom, i'm a professional Java software developer and home automation installer. I have a Loxone installation myself and want to make an Android application to control the various aspects of my home automation (security cameras, multi room audio & Loxone). I came across this library and it seems promising, but not finished as stated in the Readme. I would love to contribute to this project. Some things i miss right now are:

- Room filtering of controls (to filter controls per room for wall tablet)
- More type of controls (eg. LightingcontrollerV2, ...)

This library seems to be a good fit to use in my Android application instead of writing it myself.
